src/app/controllers/components/ControllerHeaderForms/ControllerHeaderForms.test.tsx

```diff
@@ -10,41 +7,40 @@ import {
   controller as controllerFactory,
   rootState as rootStateFactory,
 } from "testing/factories";
-
-const mockStore = configureStore();
+import { renderWithBrowserRouter } from "testing/utils";
 
 describe("ControllerHeaderForms", () => {
-  it("can render the action form wrapper", () => {
+  it("can render an action form", () => {
     const state = rootStateFactory();
-    const store = mockStore(state);
-    const wrapper = mount(
-      <Provider store={store}>
-        <MemoryRouter>
-          <ControllerHeaderForms
-            controllers={[controllerFactory()]}
-            headerContent={{ view: ControllerHeaderViews.SET_ZONE_CONTROLLER }}
-            setHeaderContent={jest.fn()}
-          />
-        </MemoryRouter>
-      </Provider>
+    renderWithBrowserRouter(
+      <ControllerHeaderForms
+        controllers={[controllerFactory()]}
+        headerContent={{ view: ControllerHeaderViews.SET_ZONE_CONTROLLER }}
+        setHeaderContent={jest.fn()}
+      />,
+      { state }
     );
-    expect(wrapper.find("ControllerActionFormWrapper").exists()).toBe(true);
+
+    expect(
+      screen.getByText(/Cannot set zone of 1 controller/)
```

Comment:
It might be better to either update the state in this test so that the form actually gets rendered (instead of the error message). Alternatively change the test description so that it reflects what's being tested here (this test case checks that a warning is displayed, the form won't show up here).